src/services/codefixes/fixOverrideModifier.ts

```diff
@@ -20,23 +20,24 @@ namespace ts.codefix {
         Diagnostics.This_parameter_property_must_be_rewritten_as_a_property_declaration_with_an_override_modifier_because_it_overrides_a_member_in_base_class_0.code
     ];
 
-    const errorCodeFixIdMap: Record<number, [DiagnosticMessage, string | undefined, DiagnosticMessage | undefined]> = {
+    const errorCodeFixIdMap: Record<number, [message: DiagnosticMessage, fixId: string, fixAllMessage: DiagnosticMessage, allowInjs: boolean]> = {

```

Comment:
Please convert this into an object type that has the same property names you just added to the tuple.